frontend/src/metabase/search/components/InfoText/InfoTextEditedInfo.tsx

```diff
@@ -0,0 +1,102 @@
+/* eslint-disable react/prop-types */
+import dayjs from "dayjs";
+import { t } from "ttag";
+import { isNull } from "underscore";
+import type { UserListResult } from "metabase-types/api";
+import { useUserListQuery } from "metabase/common/hooks/use-user-list-query";
+import Tooltip from "metabase/core/components/Tooltip";
+import { isNotNull } from "metabase/core/utils/types";
+import { getRelativeTime } from "metabase/lib/time";
+import type { WrappedResult } from "metabase/search/types";
+import { Text } from "metabase/ui";
+import { LastEditedInfoText, LastEditedInfoTooltip } from "./InfoText.styled";
```

Comment:
We should move the styled components too:

```suggestion
import { LastEditedInfoText, LastEditedInfoTooltip } from "./InfoTextEditedInfo.styled";
```